_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+{% endfigure %}
+{% endcolumn %}
+{% column right:"hxCol-6" %}
+{% figure [caption:"End date selected, range is complete"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-date-ranges-using-a-single-field-3.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Ranges spanning months or years
+If the user selects a start date in one month or year, and an end date in another month or year, all dates in-between highlight as expected.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Start date selected in October"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-ranges-spanning-months-or-years-1.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"End date selected in December"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-ranges-spanning-months-or-years-2.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Month picker
+<div class="hxRow">
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+The month picker enables users to select a different month from the entire year so that they don’t have to use the back and next arrows to traverse months.
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"User hovers over header button"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-month-picker-1.svg)
+{% endfigure %}
+{% endcolumn %}
+{% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"On click, the month picker slides down"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-month-picker-2.svg)
+{% endfigure %}
+{% endcolumn %}
+{% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"User chooses a month, and the picker slides up"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-month-picker-3.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Month picker specifications
+Month picker state maintains the same spacing considerations from Date selector state, with slight modifications of color values.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
```

Comment:
Any caption?

Reply:
Applies to all images

Reply:
I didn't think captions were necessary for most images in this doc.

Reply:
Fair enough; just wanted to ensure that's kosher as we have them nearly on all images elsewhere